clouddriver-openstack/src/main/groovy/com/netflix/spinnaker/clouddriver/openstack/client/OpenstackClientProvider.groovy

```diff
@@ -52,6 +65,127 @@ abstract class OpenstackClientProvider {
   }
 
   /**
+   * Gets a load balancer pool for a given region and load balancer UUID.
+   * @param region
+   * @param loadBalancerId
+   * @return
+   */
+  LbPool getLoadBalancerPool(final Region region, final String loadBalancerId) {
```

Comment:
I am going to move forward with the region stuff in this PR ... Don't want to pull that it out unless we have problems with the design.